pkg/state/cluster.go

```diff
@@ -242,7 +244,17 @@ func (h *Host) WorkerHealthy() bool {
 }
 
 func (h *Host) healthy() bool {
-	return h.IsInCluster && h.ContainerRuntime.Healthy() && h.Kubelet.Healthy()
+	var crStatus bool
+
+	if h.ContainerRuntimeContainerd.IsProvisioned() {
```

Comment:
I don't understand this check. How do we know that Docker is installed and supposed to be healthy if we just check is containerd provisioned? Shouldn't we check here is Docker provisioned instead?
```suggestion
	if h.ContainerRuntimeDocker.IsProvisioned() {
```

Reply:
Yep, you're absolutely right, looks like I've messed up when was moving the code around. Nice catch!